package/src/steps/FormView/BodyFormView.tsx

```diff
@@ -197,18 +197,18 @@ const BodyFormView: React.FC<BodyFormViewType> = (props) => {
     return (
```

Comment:
I just realised only the component usage was updated here, but if you can easily do this with a copy paste, I still think it would be good based on the Boy scout rule of leaving code better than how it was found

Reply:
yeah but I'm afraid that this can cause many issues when trying to merge it with dev later.
i've suggested a refactor for this a while back and maybe will do it as so the merge will get more easy to do

Reply:
https://github.com/onramper/widget/pull/24/commits/c0e1475cf16ef6f95ccf66f0f0f4e6c5917c85e5